tests/test_multipart_helpers.py

```diff
@@ -555,10 +555,10 @@ def test_attfncontqs(self) -> None:
 
     def test_attfncontenc(self) -> None:
         disptype, params = parse_content_disposition(
-            "attachment; filename*0*=UTF-8" 'foo-%c3%a4; filename*1=".html"'
+            "attachment; filename*0*=UTF-8" + 'foo-%c3%a4; filename*1=".html"'
         )
         assert "attachment" == disptype
-        assert {"filename*0*": "UTF-8" "foo-%c3%a4", "filename*1": ".html"} == params
+        assert {"filename*0*": "UTF-8" + "foo-%c3%a4", "filename*1": ".html"} == params
```

Comment:
Don't think this one benefits from the concatenation:
```suggestion
        assert {"filename*0*": "UTF-8foo-%c3%a4", "filename*1": ".html"} == params
```